### DISCUSSION TITLE
Linker Error with Bazel Dependency on Google Test

### DISCUSSION THREAD

**Discussion by jiegger** [google/googletest#4619] [Community Help]
I am facing a linker error when trying to use Google Test in my Bazel project. I have the following dependencies set up in my project:
```
deps = [
    "@googletest//:gtest",
    "@googletest//:gtest_main",
]
```
However, I encounter the following error during the linking process:
```
test/include/gtest/gtest.h:1364: error: undefined reference to 'testing::internal::EqFailure(char const*, char const*, std::string const&, std::string const&, bool)'
```
This indicates that the function testing::internal::EqFailure from gtest/internal/gtest-internal.h is not found.

I have checked my BUILD file and ensured that the dependencies are correctly set, but the issue persists.

Could anyone provide guidance on how to resolve this linker error when using Google Test with Bazel? Thank you!

**Comment by lhycms** [comment]
Have you solved this problem?
